src/App.js

```diff
@@ -67,8 +64,10 @@ class App extends Component {
             <ProteinViewer
               aminoAcids={visibleProteins[0].aminoAcids}
               svgImage={visibleProteins[0].svgImage}
+              alleles={visibleProteins[0].alleles}
```

Comment:
I ended up passing around both `aminoAcids` and `alleles`, which might be a bad idea because of redundancy. Once we're defining proteins by DNA instead of amino acids, I would consider relying only on DNA instead.

Reply:
:+1: but it doesn't need to be in this PR.